hack/local-up-cluster.sh

```diff
@@ -1112,6 +1134,14 @@ if [[ "$DEFAULT_STORAGE_CLASS" = "true" ]]; then
   create_storage_class
 fi
 
+if [[ ${FEATURE_GATES} == *"CSIDriverRegistry=true"* ]]; then
+  create_csidriver
+fi
+
+if [[ ${FEATURE_GATES} == *"CSINodeInfo=true"* ]]; then
+  create_csinodeinfo
+fi
+
```

Comment:
I think it wouldn't install the CRDs if `FEATURE_GATES` is set to `AllAlpha=true`, right?

Reply:
Would it be better to do something like this instead?

```if [[ "${FEATURE_GATES:-}" =~ "CSIDriverRegistry=true" || (...)```

Reply:
Yes, I'll change it.  Thanks!